package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "tarkov.dev",
-  "version": "2.0.2",
+  "version": "0.0.0",
```

Comment:
Why no more version?

Reply:
We don't have any [releases / tags](https://github.com/the-hideout/tarkov-dev/releases) published on GitHub, so it feels odd to track them in our `package.json` file when we don't use versions for anything in this project IMO

Reply:
Make sense 👍